pkg/proto/configuration/jwt/jwt.proto

```diff
@@ -87,5 +102,5 @@ message AuthorizationHeaderParserConfiguration {
   // extracted from the token:
```

Comment:
I think the example above needs to be changed to use `payload.given_name` and `payload.family_name`?